common/djangoapps/student/management/commands/repair_internal_admin_instructor_role.py

```diff
@@ -0,0 +1,139 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ObjectDoesNotExist
+
+from student.models import CourseAccessRole
+from django.contrib.auth.models import Group
+from edx_solutions_api_integration.models import CourseGroupRelationship
+from edx_solutions_api_integration.courseware_access import get_course_key
+from courseware import courses
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = '''Remove instructor role of internal admins on courses in their organization and add it on internal tagged courses
+example:
+    ./manage.py lms repair_internal_admin_instructor_role dryrun --settings={aws, devstack}
+
+Options:
+    createlog
+                don't print all the effects but create log file
+    dryrun
+                do the dry run for the fixes
+    repair
+                repair all roles
+'''
+
+    def handle(self, *args, **options):
+        if len(args) < 1 or len(args) > 2:
+            print Command.help
+            return
+
+        dry_run = True
+        create_log = False
+
+        msg_string = 'Script started on {}'.format(datetime.datetime.now().ctime())
+
+        if 'repair' in args:
+            dry_run = False
+        if 'dryrun' in args:
+            dry_run = True
+        if 'createlog' in args:
+            create_log = True
+            file_handler = logging.FileHandler('repair_internal_admin_instructor_role.log')
```

Comment:
@ziafazal 
Yes, but I wanted to have it in separate log file to have better control over script. If you don't want to have separate log file just don't use createlog as argument.